communication/README.md

```diff
@@ -25,9 +25,6 @@ and meetings devoted to Kubernetes.
 * [Slack] - sign up  
 
 Real time discussion at kubernetes.slack.io:   
-Discussions on most channels are archived at [kubernetes.slackarchive.io].
```

Comment:
@idvoretskyi can we also remove the `Real time discussion at kubernetes.slack.io:` line above?

Reply:
https://github.com/kubernetes/community/blame/5b4146f10df50b6255a71144efd6b87bbeea8eb8/communication/README.md#L99 too.